app/views/shared/_display_ad.html.erb

```diff
@@ -4,7 +4,12 @@
   data-category-impression="<%= DisplayAdEvent::CATEGORY_IMPRESSION %>"
   data-context-type="<%= data_context_type %>">
   <div class="crayons-sponsorship__header relative">
-    <div class="crayons-sponsorship__title"><%= Settings::Community.community_name %></div>
+    <% if Page.exists?(slug: "billboards") && display_ad.organization_id? && display_ad.type_of == "community" %>
+      <img width="24" height="24" class="radius-default crayons-sponsorship__image" src="<%= display_ad.organization.profile_image_url_for(length: 64) %>" alt="profile" loading="lazy" />
+      <div class="crayons-sponsorship__title pl-2 fs-s"><%= display_ad.organization.name %></div>
```

Comment:
Change pl-2 to pl-1 so that the gap between text/org-name and icon is 8px. Currently its 12px